dev-env-sub

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -e
```

Comment:
Can we move this file into the directory with the other scripts/testing stuff? `./relayer/examples/demo/`